src/auth/auth-get-domain.ts

```diff
@@ -0,0 +1,33 @@
+const thirdPartyDomainSTSUrlMappings: { [domain: string]: string; } = {
+  'bbk12.com': 'https://sts-sso.bbk12.com'
+};
+const defaultSTSUrl = 'https://s21aidntoken00blkbapp01.nxt.blackbaud.com';
+
+export class BBAuthGetDomain {
+
+  private static cacheDictionary: any = {};
+
+  public static isRegisteredDomain(domain: string = window.location.hostname): boolean {
+    if (domain in this.cacheDictionary) {
+      return this.cacheDictionary[domain].isRegistered;
+    }
+    let isRegistered = false;
+    Object.keys(thirdPartyDomainSTSUrlMappings).forEach((d) => {
+      let domainEndingCompare = d;
+      if (!domainEndingCompare.startsWith('.')) {
+        domainEndingCompare = '.' + domainEndingCompare;
+      }
+      if (domain === d || domain.endsWith(domainEndingCompare)) {
```

Comment:
Change this to append the `.` when doing the second comparison. Also change `endsWith()` to use `indexOf()` since `endsWith()` is not supported in IE.

```
domainEndingCompare = `.{$d}`;

if (domain === d || domain.substr(-domainEndingCompare.length) === d) {
  return d;
}
```